azure/src/main/scala/quasar/blobstore/azure/Azure.scala

```diff
@@ -57,6 +58,7 @@ object Azure extends Logging {
             .clientId(clientId.value)
             .tenantId(tenantId.value)
             .clientSecret(clientSecret.value)
+            .tokenRefreshOffset(Duration.ofMinutes(60))
```

Comment:
Curious, so the default of 2 minutes gave problems?

I'm wondering if 60 mins could be too high. As in that it may interfere with the token expiry time - say, if the token expires in 30 mins then it may end up refreshing continuously. 

Seems like the token expiry time should be higher than an hour normally, but good to make sure.

Reply:
So this is an attempt to fix this issue

```
"com.microsoft.azure.storage.blob.StorageException: Status code 403, \"\ufeff<?xml version=\"1.0\" encoding=\"utf-8\"?><Error><Code>AuthenticationFailed</Code><Message>Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.\nRequestId:4519ad75-a01e-0010-4743-1eef24000000\nTime:2020-04-29T16:29:02.3058787Z</Message><AuthenticationErrorDetail>Lifetime validation failed. The token is expired.</AuthenticationErrorDetail></Error>\""```

We haven't been able to confirm yet that this change resolves the issue but this change doesn't seem to break anything either.

Reply:
I can see how version 1.0.0 has this problem because that doesn't have `tokenRefreshOffset` yet so it basically works as if this is set to 0. 

However, I thought it may be sufficient to only upgrade the lib. This should set the `tokenRefreshOffset` to the default of 2 mins, right?
If `tokenRefreshOffset` needs to be higher than that then something must be terribly wrong with the network.

But not that opinionated about it, feel free to merge as-is if you prefer.